Dockerfile

```diff
@@ -1,10 +1,17 @@
-FROM node:16-slim as node-build
+FROM node:19-slim as node-build
 
 ARG NPM_CONFIG__AUTH
 ARG NPM_CONFIG_REGISTRY=https://workivaeast.jfrog.io/workivaeast/api/npm/npm-prod/
 ARG NPM_CONFIG_ALWAYS_AUTH=true
 ARG GIT_TAG
 
+RUN reg=$(echo "$NPM_CONFIG_REGISTRY" | cut -d ":" -f 2) && \
+    echo "$reg:_auth = $NPM_CONFIG__AUTH" > /.npmrc && \
+    echo "registry = $NPM_CONFIG_REGISTRY" >> /.npmrc && \
+    echo "always-auth = true" >> /.npmrc
+ARG NPM_CONFIG_USERCONFIG=/.npmrc
```

Comment:
What is the default file (is it `/root/.npmrc`), and we just write to that instead so we don't need this extra env var?

Reply:
Definitely. There's some Rosie hijinks on this PR so I'll get this change included on my next ixbrl-viewer PR.